API-strategie-modulen/API-strategie-mod-geo/crs.md

```diff
@@ -87,11 +87,15 @@ The guiding principles for CRS support:
 - The default CRS, CRS84, is listed first in the list of supported CRSs in the API; if the consumer does not specify the CRS it is assumed it uses the default.
 - Coordinate reference systems API strategy: request/response in RD; ETRS89; CRS84; Pseudo  Mercator;
 - Consider no-regret: record in multiple much-requested CRSs instead of on-the-fly transformation;
-- Use RDNAPTRANS™ 2018 to transform RD/Amersfoort to ETRS89 (correction grid);
+- Use RDNAPTRANS™ 2018 to transform Amersfoort / RD New to ETRS89 (correction grid);
 - Presentation depending on context (e.g. user requirements);
 - Exchange format (notation) ETRS89 and WGS84 longitude latitude in decimal degrees: DD.ddddddddd (for example: `5.962376256, 52.255023450`)
 - Exchange format (notation) RD and Pseudo Mercator X Y in meters: `195427.5200 311611.8400`
-- Use a nulltransformation to transform ETRF2000 to WGS 84 unless a time-dependent transformation is needed, see the CRS Guidelines [[hr-crs]]. If a time-dependent transformation is used for WGS 84, the CRS used as realisation of WGS 84 should be supported by the API too, e.g. if the transformation between ETRF2000 and ITRF2014 at epoch 2020 is used for request/response in WGS 84 (EPSG:4326), then the API should also support ITRF2014 (EPSG:9000).
+- Use the CRS Guidelines [[hr-crs]] for coordinate transformations.
+- CRSs may be grouped into ensemble CRSs. When exchanging geometry an ensemble member CRS shall be used (instead of an ensemble CRS) where possible.
+- Use enemble member CRS (instead of an ensemble CRS) as output of coordinate transformation, where possible.
+- APIs shall support and advertise both ensemble CRSs and ensemble member CRSs if geometry is exchanged and the CRS for the geometry is a member CRS.
+- If multiple coordinate transformation steps are required than APIs shall support and advertised all CRSs that are output of each CRS transformation.
```

Comment:
```suggestion
- If multiple coordinate transformation steps are required than APIs shall support and advertise all CRSs that are output of each CRS transformation step.
```

Reply:
Ik heb deze zin herschreven zodat deze overeenkomt met wat er in #508 en par. [3.3.2.2 Handreiking Gebruik coördinaatreferentiesystemen bij uitwisseling en visualisatie van geo-informatie](https://docs.geostandaarden.nl/crs/crs/#transformatie-tussen-etrs89-en-itrs-wgs-84) staat.